spec/models/user_spec.rb

```diff
@@ -53,5 +53,21 @@
         it { is_expected.to allow_value(email).for(:email) }
       end
     end
+
+    describe 'User roles' do
+      let(:university){create :user, email: 'oxford@oxford.edu', role: :university}
```

Comment:
change variable name to `user` to make it more readable

Reply:
fixed